src/build.js

```diff
@@ -9,7 +9,12 @@ var defaults = require('./defaults.js'),
     solveOutput = require('./solveOutput.js'),
     createCompiler = require('./createCompiler.js'),
     createLogger = require('./createLogger.js'),
-    _ = require('lodash');
+    _ = require('lodash'),
+    exitStatus = 0;
```

Comment:
Maybe it makes more sense to exit right away in this instance. Conceptually when I was think about setting a status code I don't always want to exit the process at that point. I just want to note that something went wrong and continue on, doing cleanup or retry (and setting it back to 0 if successful)

Anyway I can change the pr to exit right away upon finding an error.